### PR TITLE
added link to my profile in navbar and user profile in user list items

### PR DESCRIPTION
**Pull request by estelle-rcr**
# Description

Please provide a detailed description of what was done in this PR.
Precise the issue that you are resolving.

# Changes include

- [X] New feature (non-breaking change that adds functionality)
- [ ] Bugfix (non-breaking change that solves an issue)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it.

# Checklist

- [X] I have assigned this PR to myself
- [X] I have added at least 1 reviewer
- [X] I have added the needed labels
- [X] I have linked this PR to an issue
- [X] I have linked this PR to a milestone
- [X] I have linked this PR to a project
- [X] I have tested this code
- [ ] I have added / updated tests (unit / functionals / end-to-end / ...)
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it.

### CODE REVIEW

File: frontend/app/src/components/global-components/navbar.tsx [hunk 1]
```diff
@@ -182,7 +184,7 @@ function Navbar({ text, avatarImg }: BannerProps) {
             </DropDownMenu>
```
Comment (llescure) [inline frontend/app/src/components/global-components/navbar.tsx:184]: Remove Profile from dropDownMenu in navbar

File: frontend/app/src/components/section-components/users-list-item.tsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ import {
 import { useState } from 'react';
 import FriendsOptions from './profile/friends-options';
 import MembersOptions from './chat/members-options';
```
Comment (llescure) [inline frontend/app/src/components/section-components/users-list-item.tsx:12]: Remove Go to profile in dropdown button in member list and friends list

**Review by llescure — COMMENTED** [review]
Add link on image in match history

**Review by llescure — COMMENTED** [review]
Add link to profile on user image in ranking

File: frontend/app/src/components/global-components/navbar.tsx (hunk 1 above)
Comment (estelle-rcr) [reply frontend/app/src/components/global-components/navbar.tsx:184]: ahh je l'aurais bien gardé en plus celui-ci

**Comment by estelle-rcr** [comment]
Toutes les remarques sont intégrées sauf pour match history où on n'envoie que les données d'img des deux utilisateurs donc je voulais pas trop touché au composant pour faire descendre plus de données (id des opposants)

**Review by llescure — APPROVED** [review]

**Pull request merged**